wagtail/snippets/tests/test_bulk_actions/test_bulk_delete.py

```diff
@@ -65,7 +64,7 @@ def test_delete_with_limited_permissions(self):
 
         html = response.content.decode()
         self.assertInHTML(
-            f"<p>You don't have permission to delete these {capfirst(self.snippet_model._meta.verbose_name_plural)}</p>",
+            "<p>You don't have permission to delete these standard snippets</p>",
```

Comment:
As an aside - this is one reason why I have a pet hate for people needlessly using calculated expressions in test assertions, rather than checking for the literal result. If the writer of this test had actually written out `<p>You don't have permission to delete these Standard snippets</p>`, it might have made them pause and think "maybe capfirst isn't the output we want" :-)